taxi/admin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Car, Manufacturer, Driver
+
+admin.site.register(Manufacturer)
```

Comment:
it's better to move it to the end of file after all classes as we do in real projects